orderer/common/server/main.go

```diff
@@ -241,7 +241,9 @@ func Main() {
 	}
 	ab.RegisterAtomicBroadcastServer(grpcServer.Server(), server)
 	logger.Info("Beginning to serve requests")
-	grpcServer.Start()
+	if err := grpcServer.Start(); err != nil {
```

Comment:
Thanks for your guidance . My purpose is  improving code quality  and I hope the code will be playing a role in the community in a simple way

Reply:
Thank you for addressing the comment. I do have a couple of concerns with how the message is formatted. Apologies in advance.

First, the message includes the address of the listener instead of the name or description of the server. While the address can be used to determine which server it is, I think it may be clearer to simply say something like `Atomic Broadcast gRPC server has terminated: %v`. It indicates that the server is the the one we register for the common ordering endpoint and not the cluster server.

Second, the use of `[` and `]` around messages is far too common in Fabric and, in may cases, confusing. In this particular case, since we're providing an address, it also matches the common formatting for IPv6 addresses and the one required when URL encoding. (example http://[::1]:8080/).

So, while improved, I would like to have the brackets dropped and, if possible, use static text instead of the address in the message.

Thanks.

Reply:
Thank you for your patience and guidance. Thanks.
Could you help me review the code again ? @sykesm